### ISSUE TITLE
User should be able to sync with other integrations

### ISSUE THREAD

**Issue by pfych** [TNG-dev/tachi-bot#7]
As a user I should be able to `/sync` and tachi will sync my scores with linked integrations

**Comment by zkldi** [comment]
More notes on this:

```
/sync <FLO|EAG|MIN> <iidx|sdvx>
```
Should make a POST request of
```
https://tachi.url/api/v1/import/from-api
---
{
	"importType": "api/flo-iidx", // or flo-sdvx, eag-iidx, eag-sdvx, min-sdvx
}
```

Note that there is no `min-iidx`.

As long as this request is properly authorised (and your keys have `submit_score`, they will) this will Just Work(tm).

**Comment by zkldi** [comment]
Fixed in #46.
